tutorials/fermyon-spin-rust-setup-guide/index.md

```diff
@@ -0,0 +1,464 @@
+---
+description: Setup guide for configuring Turso for deployment using Fermyon Spin and Rust.
+keywords:
+  - turso
+  - tutorial
+  - guide
+  - fermyon
+  - spin
+---
+
+# Fermyon Spin with Rust setup guide
+
+In this setup guide, you will deploy a simple route component to Fermyon Spin
+that queries a [Turso] database. The deployment is configured with variables
+whose values are obtained from the [Turso CLI].
+
+## Prerequisites
+
+- A [Fermyon Cloud account]
+- The Turso CLI installed on your machine ([installation instructions])
+- Rust toolchain installed [using rustup] (Homebrew installation will not work)
+- Spin CLI installed
+- Familiarity with Spin is suggested ([follow the Spin quickstart])
+
+Check that the turso CLI is installed:
+
+```
+$ turso --version
+```
+
+Check that rust is installed using rustup:
+
+```
+$ rustup --version
+rustup 1.26.0 (5af9b9484 2023-04-05)
```

Comment:
Is the version of rustup really important?
I'd use `rustup show` instead here, and it will also show you that you have a stable toolchain.

Reply:
The version is likely not important. That's just normally what I use to verify that something is installed, and a baseline for understanding what the creator of the tutorial was looking at when it was created.

`rustup show` contains a lot more information that also not really relevant to everyone. Is there something else that just gives a quick status check to make sure things are OK?

Reply:
`rustup --version` confuses me, because it is such wildly different version than the rust toolset itself. They do print a warning saying that is the case, but before I got to it my brain stopped for a minute in "how is it that Doug has such an ancient version?"

I think it's fine, though.